server/webhook_jira_test.go

```diff
@@ -0,0 +1,126 @@
+		},
+		"updated labels": {
+			TestWebhook:    getJiraTestData("webhook-issue-updated-labels.json"),
+			ExpectedEvents: []string{"event_updated_labels", "event_updated_all"},
+		},
+		"lowered priority": {
+			TestWebhook:    getJiraTestData("webhook-issue-updated-lowered-priority.json"),
+			ExpectedEvents: []string{"event_updated_priority", "event_updated_all"},
+		},
+		"raised priority": {
+			TestWebhook:    getJiraTestData("webhook-issue-updated-raised-priority.json"),
+			ExpectedEvents: []string{"event_updated_priority", "event_updated_all"},
+		},
+		"updated rank": {
+			TestWebhook:    getJiraTestData("webhook-issue-updated-rank.json"),
+			ExpectedEvents: []string{"event_updated_rank", "event_updated_all"},
+		},
+		"updated summary": {
+			TestWebhook:    getJiraTestData("webhook-issue-updated-renamed.json"),
+			ExpectedEvents: []string{"event_updated_summary", "event_updated_all"},
+		},
+		"reopened issue": {
+			TestWebhook:    getJiraTestData("webhook-issue-updated-reopened.json"),
+			ExpectedEvents: []string{"event_updated_reopened", "event_updated_status", "event_updated_all"},
+		},
+		"resolved issue": {
+			TestWebhook:    getJiraTestData("webhook-issue-updated-resolved.json"),
+			ExpectedEvents: []string{"event_updated_resolved", "event_updated_status", "event_updated_all"},
+		},
+		"updated sprint": {
+			TestWebhook:    getJiraTestData("webhook-issue-updated-sprint.json"),
+			ExpectedEvents: []string{"event_updated_sprint", "event_updated_all"},
+		},
+		"updated status": {
+			TestWebhook:    getJiraTestData("webhook-issue-updated-started-working.json"),
+			ExpectedEvents: []string{"event_updated_status", "event_updated_all"},
+		},
+		"CLOUD comment created": {
+			TestWebhook:    getJiraTestData("webhook-cloud-comment-created.json"),
+			ExpectedEvents: []string{"event_created_comment"},
+		},
+		"CLOUD comment deleted": {
+			TestWebhook:    getJiraTestData("webhook-cloud-comment-deleted.json"),
+			ExpectedEvents: []string{"event_deleted_comment"},
+		},
+		"CLOUD comment updated": {
+			TestWebhook:    getJiraTestData("webhook-cloud-comment-updated.json"),
+			ExpectedEvents: []string{"event_updated_comment"},
+		},
+		"SERVER comment created": {
+			TestWebhook:    getJiraTestData("webhook-server-issue-updated-commented-1.json"),
+			ExpectedEvents: []string{"event_created_comment"},
+		},
+		"SERVER comment deleted": {
+			TestWebhook:    getJiraTestData("webhook-server-issue-updated-comment-deleted.json"),
+			ExpectedEvents: []string{"event_deleted_comment"},
+		},
+		"SERVER comment updated": {
+			TestWebhook:    getJiraTestData("webhook-server-issue-updated-comment-edited.json"),
+			ExpectedEvents: []string{"event_updated_comment"},
+		},
+		"SERVER comment created notify": {
+			TestWebhook:    getJiraTestData("webhook-server-issue-updated-commented-2.json"),
+			ExpectedEvents: []string{"event_created_comment"},
+		},
+		"SERVER comment created ignored": {
```

Comment:
@levb I've "ignored" this event the same way it is being ignored in [webhook_parser.go](https://github.com/mattermost/mattermost-plugin-jira/blob/7109b86e183ca357d6019c86f5dee17a11db6c85/server/webhook_parser.go#L53)